example-scripts/powershell/publishExisting.ps1

```diff
@@ -0,0 +1,21 @@
+#Powershell sees curly brackets as codeblocks so use quotations marks "{}" or just write your guids without them
+$componentGuids =""
+
+$componentFiles = Get-ChildItem *.mpms
```

Comment:
Is there a better way to get this information?
E.g. by looking into the EA model and extracting all MPMS definitions with a specific stereotype?
Or maybe by having a json config controlling the export. The MPMS content will have to be changed after each model change and therefore be rarely consistent with the EA model when maintained in GIT. Or we have to add after each push a commit "Updated MPMS" with a workflow

Reply:
We need this for a project now - I guess